hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMaster.java

```diff
@@ -520,7 +520,7 @@ public HMaster(final Configuration conf)
 
       this.rsFatals = new MemoryBoundedLogMessageBuffer(
           conf.getLong("hbase.master.buffer.for.rs.fatals", 1 * 1024 * 1024));
-      LOG.info("hbase.rootdir=" + getRootDir() +
+      LOG.info("hbase.rootdir=" + getDataRootDir() +
```

Comment:
Should do parametrized logging if you are making changes... i.e. "hbase.rootdir={}..."

Reply:
getDataRootDir is diff from getRootDir?

Reply:
This is part of
> Rename `fs` to `dataFs`, `rootDir` as `dataRootDir` so as to distinguish from the new `walFs`, `walRootDir` (and make it easier to spot bugs).

I think I see some bugs...